project_national/housing_characteristics/Refrigerator.tsv

```diff
@@ -1,24 +1,11 @@
-Dependency=Geometry Building Type RECS	Dependency=Usage Level	Option=EF 10.2	Option=EF 10.5	Option=EF 15.9	Option=EF 17.6	Option=EF 6.7	sample_count	sample_weight
-Mobile Home	Average	0.024380	0.066827	0.321826	0.562587	0.024380	133.250000	1710517.000000
-Multi-Family with 2 - 4 Units	Medium	0.013683	0.073747	0.322583	0.576304	0.013683	230.250000	2243989.500000
-Multi-Family with 5+ Units	Low	0.014750	0.061490	0.300636	0.607870	0.014727	475.000000	4721506.000000
-Single-Family Attached	High	0.015515	0.066510	0.281868	0.620327	0.015502	221.750000	1674581.625000
-Single-Family Detached	Average	0.012364	0.069471	0.256486	0.648904	0.012347	1935.750000	17833094.000000
-Mobile Home	Medium	0.024380	0.066827	0.321826	0.562587	0.024380	133.250000	1710517.000000
-Multi-Family with 2 - 4 Units	Low	0.013683	0.073747	0.322583	0.576304	0.013683	230.250000	2243989.500000
-Multi-Family with 5+ Units	High	0.014750	0.061490	0.300636	0.607870	0.014727	475.000000	4721506.000000
-Single-Family Attached	Average	0.015515	0.066510	0.281868	0.620327	0.015502	221.750000	1674581.625000
-Single-Family Detached	Medium	0.012364	0.069471	0.256486	0.648904	0.012347	1935.750000	17833094.000000
-Mobile Home	Low	0.024380	0.066827	0.321826	0.562587	0.024380	133.250000	1710517.000000
-Multi-Family with 2 - 4 Units	High	0.013683	0.073747	0.322583	0.576304	0.013683	230.250000	2243989.500000
-Multi-Family with 5+ Units	Average	0.014750	0.061490	0.300636	0.607870	0.014727	475.000000	4721506.000000
-Single-Family Attached	Medium	0.015515	0.066510	0.281868	0.620327	0.015502	221.750000	1674581.625000
-Single-Family Detached	Low	0.012364	0.069471	0.256486	0.648904	0.012347	1935.750000	17833094.000000
-Mobile Home	High	0.024380	0.066827	0.321826	0.562587	0.024380	133.250000	1710517.000000
-Multi-Family with 2 - 4 Units	Average	0.013683	0.073747	0.322583	0.576304	0.013683	230.250000	2243989.500000
-Multi-Family with 5+ Units	Medium	0.014750	0.061490	0.300636	0.607870	0.014727	475.000000	4721506.000000
-Single-Family Attached	Low	0.015515	0.066510	0.281868	0.620327	0.015502	221.750000	1674581.625000
-Single-Family Detached	High	0.012364	0.069471	0.256486	0.648904	0.012347	1935.750000	17833094.000000
+Dependency=Geometry Building Type RECS	Option=EF 6.7	Option=EF 10.2	Option=EF 10.5	Option=EF 15.9	Option=EF 17.6	Option=EF 19.9	Option=None	sample_count	sample_weight
```

Comment:
These two files are different. I am guessing that the removal of Usage Level and the added EF 19.9 option is the change we want to keep as the efficiencies I think have been updated.

Reply:
In this case, resstock-estimation has the correct file.

Reply:
@afontani I think the "None" option needs to be removed, as that option does not exist in the options lookup. Can you show me what needs to be done in the RECS 2009 tsv maker?